<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlWidget</class>
 <widget class="QWidget" name="ControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>245</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>182</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="AxisControlWidget" name="axis0Control" native="true"/>
     </item>
     <item>
      <widget class="AxisControlWidget" name="axis1Control" native="true"/>
     </item>
     <item>
      <widget class="AxisControlWidget" name="axis2Control" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AxisControlWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/AxisControlWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
